examples/example-gateway/endpoints/tchannel/baz/baz_call.go

```diff
@@ -63,9 +61,10 @@ func (w CallEndpoint) Handle(
 		case *clientBaz.AuthErr:
 			return respHeaders, (*endpointBaz.AuthErr)(v)
 		default:
-			w.Logger.Error(
+			zanzibar.OnError(
```

Comment:
nitpick: `zanzibar.LogError()`

Reply:
Oh I see ; maybe call this method `LogErrorWarn()` ?

Reply:
Sure calling it `LogErrorWarn`.